### PR TITLE
Fix Dasdboard not show Ticket ID , Message

### PR DESCRIPTION
**Pull request by MingjunGeng**
## Description

When logged in, Ticket ID, Message is not displayed on Dashboard>>>Tickets Dashboad. I modified /src/compone by
nts/pages/Dashboard/Dashboard.js fix bug

#### Video Link

[Loom Video](https://loom.com/share/f723af8614b546d983dcea51cd42a71d)

#### Trello Link

https://trello.com/c/mIQ9nSLR/394-bugdashboard-not-show-ticket-id-message

(Example below):

<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;JfipwNkb&#x2F;265-fix-refactor-sidebar-to-have-icons-representing-each-element-in-the-sidebar">Fix: Refactor sidebar to have icons representing each element in the sidebar</a></blockquote>

## Type of change

___Please delete options that are not relevant.___

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by FftyShadesofCode — APPROVED** [review]
Looks good!

**Review by fabiolavelasco — APPROVED** [review]
Great job!

File: src/components/pages/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -81,31 +81,33 @@ const Dashboard = props => {
   }, []);
 
   const data = [];
-
+  const escaTickets = tickets.filter(x => x.ticket_status == 'approved');
```
Comment (gastonirby) [inline src/components/pages/Dashboard/Dashboard.js:84]: I think you should change your double equals to a triple to get rid of the warning.

**Review by gastonirby — DISMISSED** [review]
Overall a good job! I just would change your equals sign on line 84, but that's a simple change!

File: src/components/pages/Dashboard/Dashboard.js (hunk 1 above)
Comment (MingjunGeng) [reply src/components/pages/Dashboard/Dashboard.js]: Thanks, this is  better than before.

**Pull request merged**